frontend/src/components/app/app.tsx

```diff
@@ -21,6 +21,8 @@ import { Dashboard } from 'components/dashboard/dashboard';
 import { Interview } from 'components/interview/interview';
 import { Interviews } from 'components/interviews/interviews';
 import { NotFound } from 'components/not-found/not-found';
+import { StudentCourse } from 'components/student-course/student-course';
+import { StudentCourseModule } from 'components/student-course-module/student-course-module';
```

Comment:
```suggestion
```
we don't want to create new pages. it should be a part of the `course` and `module` page. as `configure-group` (create) and `configure-group/:id` (edit)